app/presenters/idv/in_person/verification_results_email_presenter.rb

```diff
@@ -47,6 +47,10 @@ def service_provider_homepage_url
         sp_return_url_resolver.homepage_url if service_provider
       end
 
+      def needs_proof_of_address?
+        !(enrollment.current_address_matches_id || enrollment.capture_secondary_id_enabled)
+      end
```

Comment:
I don't know the answer to this at this time but I am wondering if this function (or the function above) can be shared? The function is same and for maintainability wondering if one gets updated without the other- would that cause a gnarly bug or likely that'd be okay? Maybe it cannot be shared. Just something I am curious about.

Reply:
@gina-yamada A refactor would be out of scope for this ticket, but your point is well taken. I think we should more broadly think about how we can DRY-up our code. Another thing to consider is that this particular piece of code will likely change once DAV is no longer behind a feature flag.